scripts/tools/accessControl/ApplyRules.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\tao\scripts\tools\accessControl;
+
+use oat\oatbox\extension\script\ScriptAction;
+use oat\tao\model\accessControl\func\AclProxy;
+use oat\tao\model\accessControl\func\AccessRule;
+
+class ApplyRules extends ScriptAction
```

Comment:
```suggestion
class ApplyActionRules extends ScriptAction
```

Do we plan to have the ACL for other purposes than Action here?

Reply:
This tool is able to apply any mask: whole extension, specified controller or specified action.
I don't think we should rename it.

Reply:
Ok understood :) 

Maybe we could also write an example with this case in the php docs then? Wdyt?